Tasks/DotNetCoreCLI/dotnetcore.ts

```diff
@@ -77,7 +78,9 @@ export class dotNetExe {
             }
             dotnet.line(dotnetArguments);
             try {
-                var result = await dotnet.exec();
+                var result = await dotnet.exec(<tr.IExecOptions>{
+                    cwd: path.dirname(projectFile)
```

Comment:
is there a possibility of regression here? I am not sure what will it mean for nuget restores etc